### PR TITLE
Proposal for Hyperledger Fabric Block Archiving

### PR DESCRIPTION
**Pull request by nekia**
The project aims to reduce the total amount of storage space required for an organisation to operate a Hyperledger Fabric network by archiving block data into a repository.

### CODE REVIEW

**Comment by lehors** [comment]
The DCO check failed and the description suddenly refers to "BlockVault" without introduction. I'm not sure whether this is leftover from an incomplete renaming or what.
Otherwise the project is definitely interesting and certainly qualifies as a lab.

Practically speaking this is implemented as a fork of Fabric. We should keep that linkage while importing the code so that changes in Fabric can be incorporated without too much hassle.

**Comment by trbs** [comment]
Agree with @lehors above :-)

Please also fill in the description of the PR.

File: labs/fabric-block-archiving.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Lab Name
+Hyperledger Fabric Block Archiving
+
+# Short Description
+
+* Reduce the total amount of storage space required for an organisation to operate a Hyperledger fabric network by archiving block data into repository.
+* For organisations to operate a Hyperledger fabric network with low resourced nodes such as a IoT edge devices.
```
Comment (tkuhrt) [inline labs/fabric-block-archiving.md]: Please change fabric to Fabric in these two sentences.

**Comment by tkuhrt** [comment]
See [steps 9-11 in this gist](https://gist.github.com/tkuhrt/10211ae0a26a91a8c030d00344f7d11b) for information that might help you to resolve the DCO issue.

**Comment by tkuhrt** [comment]
@hartm : Can you confirm your sponsorship of this lab?

**Comment by hartm** [comment]
@tkuhrt I am a sponsor of this lab, yes.

**Comment by nekia** [comment]
Thank you all for feedback. We've committed the following:
* Fix DCO issue
* Capitalize Hyperledger Fabric
* Add description of PR

We're also going to update the technical overview document, which is referred from the proposal, to replace `blockVault` to a generalized naming `block archiving`. Because some services and companies which are using the name have already existed (although I'm not sure we need to avoid this conflict).

**Comment by vipinsun** [comment]
Did not realize that Request for change is a blocker until I re-approve- above and beyond two approvers, my apologies

**Pull request merged**